Packs/Rapid7_InsightIDR/IncidentTypes/incidenttype-Rapid7_InsightIDR_Alert.json

```diff
@@ -0,0 +1,33 @@
+{
+	"id": "Rapid7 InsightIDR Alert",
+	"version": 1,
+	"modified": "2020-11-23T08:40:01.566113113+02:00",
+	"sortValues": null,
+	"packID": "",
+	"itemVersion": "",
+	"fromServerVersion": "",
+	"toServerVersion": "",
+	"vcShouldIgnore": false,
+	"vcShouldKeepItemLegacyProdMachine": false,
+	"commitMessage": "",
+	"shouldCommit": false,
+	"locked": false,
+	"name": "Rapid7 InsightIDR Alert",
+	"prevName": "Rapid7 InsightIDR Alert",
+	"color": "#B7E7FF",
+	"hours": 0,
+	"days": 0,
+	"weeks": 0,
+	"hoursR": 0,
+	"daysR": 0,
+	"weeksR": 0,
+	"system": false,
+	"readonly": false,
+	"default": false,
+	"autorun": false,
+	"preProcessingScript": "",
+	"closureScript": "",
+	"disabled": false,
+	"reputationCalc": 0,
+	"layout": "128c1d5e-0ef3-48e4-871f-c056103398cb"
```

Comment:
did not we want to create a layout with the added fields - alert ID, alert name?

Reply:
Just saw that you did, please conenct it with a more verbal id as I suggested.

Reply:
You mean to replace "layout": "128c1d5e-0ef3-48e4-871f-c056103398cb" to "layout": "Rapid7 InsightIDR - Layout"? I just want to make sure

Reply:
yes